contribute/merge-pull-request.md

```diff
@@ -31,9 +35,9 @@ See [formatting guidelines](create-pull-request.md#formatting-guidelines) for mo
 
 ### Assign a milestone
 
-It's recommended to add a milestone to every pull request. This makes it easier to track what changes did go into a certain release. Without this you're basically left with going through git commits which could be a lot harder.
+A milestone **should** be added to every pull request. Several things in the Grafana release process requires at least pull requests to be in a milestone, for example [generating changelog/release notes](#include-in-changelog-and-release-notes).
```

Comment:
```suggestion
We recommended that you add a milestone to every pull request. Several things in the Grafana release process require at least one pull request to be in a milestone, for example [generating changelog/release notes](#include-in-changelog-and-release-notes).
```

Reply:
"require at least one pull request to be in a milestone" is not true. Soon there will be a required check on pull requests that a milestone has been added so I want this to read that it's required.